apps/sms/views/conversation/js/conversation.js

```diff
@@ -3000,7 +3001,7 @@ var ConversationView = {
       type: Compose.type
     });
 
-    Drafts.add(draft);
+    Drafts.add(draft).store();
```

Comment:
It's good that we can have a sync behavior with drafts.delete that add/delete action don't call store immediately. Wondering if we could return Drafts.add(draft).store() here and utilize saveDraft() for storeDraftFromMessage/navigateToComposer ? Like this.saveDraft(draftOpt) and in saveDraft we can use the default draft option or provided draft option.